tests/testthat/test-rolling_origin.R

```diff
@@ -99,9 +99,15 @@ test_that("lag", {
       (i + attr(rs5, "initial") - attr(rs5, "lag")):(i + attr(rs5, "initial") + attr(rs5, "assess") - 1)
     )
   }
-
-  expect_error(rolling_origin(drinks, initial = 5, lag = 6)) # lag must be less than training observations
-  expect_error(olling_origin(drinks, lag = 2.1)) # lag must be whole number
+  
+  skip_if_not_installed("modeldata")
+  data("drinks", package = "modeldata", envir = rlang::current_env())
+  expect_snapshot(error = TRUE, {
+    rolling_origin(drinks, initial = 5, lag = 6)
+  })
```

Comment:
```suggestion
  expect_snapshot(error = TRUE, {
    # lag must be less than the number of training observations
    rolling_origin(drinks, initial = 5, lag = 6)
  })
```

I say this comment ought to stay around